chat-widget/src/plugins/createChatTranscript.ts

```diff
@@ -80,13 +141,33 @@ class TranscriptHTMLBuilder {
         return htmlData;
     }
 
+    createScriptElement(src: string, integrity: string | undefined = undefined, crossOrigin: string | undefined = undefined, referrerPolicy: string | undefined = undefined) {
+        return `<script src="${src}" ${integrity ? `integrity="${integrity}"`: ""} ${crossOrigin ? `crossorigin="${crossOrigin}"`: ""} ${referrerPolicy ? `referrerpolicy="${referrerPolicy}"`: ""}><\/script>`;
+    }
+
     createExternalScriptElements() {
+        const webChatScript = this.externalScripts?.botframeworkWebChat?.src?

```

Comment:
I agree this way would be cleaner, but it won't cover the scenario when `externalScript?.src` is `""`. `src` property **MUST** always be valid. Otherwise, transcript won't have all the necessary scripts to be functional.

I am still open for suggestions to on how this can be simplified for better readability